upup/pkg/fi/cloudup/apply_cluster.go

```diff
@@ -91,6 +91,8 @@ var (
 	AlphaAllowALI = featureflag.New("AlphaAllowALI", featureflag.Bool(false))
 	// CloudupModels a list of supported models
 	CloudupModels = []string{"proto", "cloudup"}
+	// OldestSupportedKubernetesVersion is the oldest kubernetes version that is supported in Kops
+	OldestSupportedKubernetesVersion = "1.9.0"
```

Comment:
When we remove code for handling old versions, we should add another cutoff below which kops should refuse to run. That should probably be what is named `OldestSupportedKubernetesVersion`. This should probably be `OldestRecommendedKubernetesVersion` or `NewestDeprecatedKubernetesVersion = 1.8.15`

I could also see a third cutoff `OldestUpgradeableKubernetesVersion` below which kops won't upgrade from.

Reply:
These are great suggestions ... we can probably do it separately?

We can recommend newer versions of k8s (and kops) in `channels/stable` ... with varying degrees of forcefulness :-)